src/parser/parser.c

```diff
@@ -176,7 +172,8 @@ int yfp_ident(struct yfcs_identifier * node, struct yf_lexer * lexer) {
             yfl_unlex(lexer, &tok);
             /* There's no prefix. */
             strcpy(node->name, node->filepath);
-            strcpy(node->filepath, lexer->input->identifier_prefix);
+            //strcpy(node->filepath, lexer->input->identifier_prefix);
```

Comment:
Keeping this in the code?

Reply:
Yeah, just in case we need it later.

Reply:
also, file tree beta looking nice
![image](https://user-images.githubusercontent.com/35516367/156873551-739f2f90-e343-4eb7-a3f0-a0e683cbf800.png)

Reply:
Yo I didn't even know that existed!

Reply:
you gotta turn on the beta tho